### DISCUSSION TITLE
Forth vs. Asm

### DISCUSSION THREAD

**Discussion by Whammo** [jkotlinski/durexforth#317] [General]
```forth
 : bneck begin chrin ?dup while emit repeat ;
 ```

                     or

```forth

code bneck
           w stx,
here
       $ffcf jsr,
+branch
             beq,
swap
       $ffd2 jsr,
             jmp,
:+
           w ldx,
;code
```
 

edit: playing with the editor

**Comment by jkotlinski** [comment]
Oh! Gotta say I way prefer Forth in this case!

**Reply by Whammo** [reply]
I think so too. They're equivalent, and for me the assembly is easier to read like this on the machine, it's still not as easy to read as the Forth.
 I suppose the assembly is faster too, but what is slow is the Kernal and the serial bus.
